Projects/RegExp-Helper-App.md

```diff
@@ -0,0 +1,40 @@
+# Regular Expression Helper
+
+**Tier:** 2-Intermediate
+
+Regular Expressions should be a valuable part of any developers toolbox. They
+provide a concise way to describe a pattern that can be used by JavaScripts
+RegExp object to test, search, match, replace, or split the contents of a 
+string. Regular Expressions provide functionality you might otherwise have to
+implement using loops and more lines of code.
+
+The Regular Expression Helper helps you learn more about Regular Expressions
+by building a useful tool you'll also be able to use to test expressions
+you use in your apps.
+
+## User Stories
+
+-   [ ] User can enter a regular expression.
+-   [ ] User can enter a string the regular expression can be run against.
+-   [ ] User can click a 'Run' button to test
+-   [ ] User can see a warning message if no regular expression was entered.
+-   [ ] User can see a warning message if no string was entered.
+-   [ ] User can see a message indicating if `test()` was able to locate the pattern in the string.
```

Comment:
Instead of a message, I believe that the User should see the “matching” text highlighted, no? What do you think?

Reply:
Agreed. Changed this story as suggested and added a new one to display a message only if there was no match.